src/System Application/App/Feature Key/src/FeatureManagementImpl.Codeunit.al

```diff
@@ -335,6 +335,28 @@ codeunit 2610 "Feature Management Impl."
         end;
     end;
 
+    /// <summary>
+    /// Returns true if the feature is enabled and data update, if required, is complete.
+    /// </summary>
+    /// <param name="FeatureId">the feature id in the system table "Feature Key"</param>
+    /// <param name="AllowInsert">specifies if inserts are allowed while checking for feature being enabled</param>
+    /// <returns>if the feature is fully enabled</returns>
+    procedure IsEnabled(FeatureId: Text[50]; AllowInsert: Boolean): Boolean;
+    var
+        FeatureKey: Record "Feature Key";
+        FeatureDataUpdateStatus: Record "Feature Data Update Status";
+    begin
+        if AllowInsert then
```

Comment:
Can we combine the two functions? If this is an overload, I'd assume the other function should've minimal code and calls this function with default parameters.

Reply:
Fair enough, refactored to be one procedure.